lua/weapons/weapon_Auto_Shotgun.lua

```diff
@@ -0,0 +1,135 @@
+-- Easy Settings part
+
+-- SWEP Info
+SWEP.Author = "yousifgaming"
+SWEP.Instructions = "Primary is the same as Secondary fire automatic shotgun"
+SWEP.Purpose = "To play with Dingus"
+SWEP.Contact = "Don't Contact me i dont like people"
+
+-- SWEP info in spawnmenu
+SWEP.PrintName = "Auto Shotgun"
+SWEP.Category = "yousifgaming's Weapons"

```

Comment:
Use `TASWeapons.Category`.

Reply:
Also use the `TASWeapons.Authors` table rather than `SWEP.Author = "yousifgaming"`